### PR TITLE
Issue #15605: update description of TODO Check

### PR DESCRIPTION
**Pull request by romani**
#15605

### CODE REVIEW

File: src/xdocs/checks/misc/todocomment.xml [hunk 1]
```diff
@@ -75,7 +75,8 @@ public class Example1 {
 }
         </source>
         <p id="Example2-config">
-          To configure the check for comments that contain <code>TODO</code> and <code>FIXME</code>:
+          To configure the check for comments that contain <code>TODO</code> and <code>FIXME</code>
+          case insensitive:
```
Comment (nrmancuso) [inline src/xdocs/checks/misc/todocomment.xml:79]:
```suggestion
          To configure the check for comments that contain <code>TODO</code> and <code>FIXME</code>,
          case-insensitive:
```

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/xdocs/checks/misc/todocomment.xml (hunk 1 above)
Comment (romani) [reply src/xdocs/checks/misc/todocomment.xml]: done

**Comment by romani** [comment]
Github, generate website

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**